config/_default/menus.yml

```diff
@@ -99,7 +99,7 @@ reference:
 
   - name: Python
     parent: Pulumi SDK
-    url: /docs/reference/pkg/python/pulumi
+    url: /docs/reference/pkg/python/providers/pulumi.html
```

Comment:
Why is it under `/providers` now? Seems odd since `pulumi` isn't a provider. Can we fix that?